_posts/2015-05-15-Rust-1.0.md

```diff
@@ -0,0 +1,331 @@
+---
+layout: post
+title: "Announcing Rust 1.0"
+author: The Rust Core Team
+---
+
+Today we are very proud to announce the
+[1.0 release of Rust][relnotes], a new programming language aiming to
+make it easier to build reliable, efficient systems. **Rust combines
+low-level control over performance with high-level convenience and
+safety guarantees**. Better yet, it achieves these goals without
+requiring a garbage collector or runtime, making it possible to
+[use Rust libraries as a "drop-in replacement" for C][ffi]. If you'd
+like to experiment with Rust, the
+["Getting Started" section of the Rust book][book] is your best bet
+(if you prefer to use an e-reader, Pascal Hertleif maintains
+[unofficial ebook vesions][ebook] as well).
+
+What makes Rust different from other languages is its type system,
+which represents a refinement and codification of "best practices"
+that have been hammered out by generations of C and C++
+programmers. As such, Rust has something to offer for both experienced
+systems programmers and newcomers alike: experienced programmers will
+find they save time they would have spent debugging, whereas newcomers
+can write low-level code without worrying about minor mistakes leading
+to mysterious crashes.
+
+### What does it mean for Rust to be 1.0?
+
+The current Rust language is the result of a lot of iteration and
+experimentation. The process has worked out well for us: Rust today is
+both simpler and more powerful than we originally thought would be
+possible. But all that experimentation also made it difficult to
```

Comment:
Not an error, but I recommend "made it difficult to maintain a Rust project" become "made it difficult to maintain projects written in Rust" for clarity.